tests/vmi_iothreads_test.go

```diff
@@ -23,7 +23,10 @@ import (
 	"context"
 	"encoding/xml"
 	"fmt"
+	"strconv"
 
+	"kubevirt.io/kubevirt/pkg/libvmi"
```

Comment:
can you separate `kubevirt.io/kubevirt/pkg` and `kubevirt.io/kubevirt/tests` imports, and order them to be last?